snap/backend/src/utils/mosaicUtils.js

```diff
@@ -0,0 +1,65 @@
+import symbolClient from '../services/symbolClient.js';
+import stateManager from '../stateManager.js';
+
+const mosaicUtils = {
+	/**
+     * Update mosaic info in state if given mosaic ids not exist in mosaicInfo.
+     * @param {object} state - The state object.
+     * @param {Array<string>} mosaicIds - The mosaic ids.
+     * @returns {Promise<void>} - A promise that resolves when mosaic info is updated.
+     */
+	updateMosaicInfo: async (state, mosaicIds) => {
```

Comment:
In what case does the mosaic id gets into the account but then not found?  Or did I misunderstand the scenario?